src/dbnode/storage/bootstrap.go

```diff
@@ -128,6 +143,8 @@ func (m *bootstrapManager) Bootstrap() (BootstrapResult, error) {
 		// reshard occurs and we need to bootstrap more shards.
 		m.hasPending = true
 		m.Unlock()
+		asyncResult.bootstrapStarted.Done()
```

Comment:
Does it make sense for consistency to also set the `asyncResult.bootstrapResult`?
```go
result := BootstrapResult{AlreadyBootstrapping: true}
asyncResult.bootstrapResult = result
asyncResult.bootstrapStarted.Done()
asyncResult.bootstrapCompleted.Done()
return result, errBootstrapEnqueued
```

Reply:
Completely missed it, will use defer for this.